odoo/addons/base/views/ir_model_views.xml

```diff
@@ -347,8 +347,13 @@
                     <field name="name"/>
                     <field name="field_description"/>
                     <field name="model_id"/>
-                    <field name="ttype"/>
+                    <field name="ttype" string="Database Type"/>
```

Comment:
Why override the field's string? `Field Type` seems correct to me

Reply:
We have a field called **Type** which is a little confused, because it says base or custom, that's why I set this string.

Reply:
I think neither is necessary, simply using Field Type + viewing the content will be explicit enough, let's not change for change's sake.

Please remove the string from the view and leave the python as is :)